### DISCUSSION TITLE
what is the use of simulated data?

### DISCUSSION THREAD

**Discussion by TropicalBreeding** [hahnlab/CAFE5#150] [Q&A] [answered]
I practiced the tutorial, and ran the Test_data (Primate_data) following 'Test_data_readMe.docx' included in the CAFE5 package. Now I understand most of functions and their logics. However one thing I may need help to understand is the simulation. I simulated and analyzed the data, including estimating lambda etc, but still wondering what I can do with this--the tutorial doesn't have any further information after the simulation.  I will greatly appreciate if anyone help me with this.

**Comment by hahnlab-user** [comment, marked as answer]
The simulation is useful when you want to compared models with different numbers of parameters. The likelihoods of these models are not chi-squared distributed, so you can simulate the null model to determine whether it is sensible to accept the model with more parameters.


matt
